pages/dashboard/my-feed.tsx

```diff
@@ -24,12 +24,10 @@ const MyFeedPage: NextPage<InitialProps> = () => {
 
   return (
     <AuthGate>
-      {(currentUser) => (
```

Comment:
@nlitwin Sorry I'm sure I'm just missing this somewhere but browsing up and down a few times I can't seem to find it, where did `currentUser` move to?

Reply:
It looks like it's just been removed from this level of my feed, but we're going to need it for various things such as their stats bar soon just as an example.

Reply:
It used to just be a `console.log(currentUser)` in `MyFeed.tsx`, which was taken out recently, so I just removed it here too. I tend to remove unused code, cause it feels weird to leave around and it's easy to add back, but I also don't have a strong preference, I can put it back if you'd like! 😄

Reply:
I generally agree but we're moving super fast right now and so there are a few things here and there that aren't perfect but I don't think particularly bad, and in this case it was left for a reason as we'd discussed previously when I removed the `console.log()`. Please put it back as I'll be using it very soon

Reply:
Also I've mentioned this a few times before but when there are random things that just get removed or changed that aren't related to the PR, I think it's really helpful to just mention them with a comment because in this case this jumped out at me and then I was reviewing the PR with the assumption that this was a part of the PR and in the back of my mind was looking for where this would connect. If I knew right away this was not related I wouldn't have been looking for that connective piece while reviewing the rest

Reply:
Cool i'll put it back! 😊 I made sure to mention it in the description under `My Feed`: `Remove render prop wrapper since currentUser is longer being used in MyFeed.tsx` that way the context was there. I can move that type of context to an actual code comment instead of the PR description in the future, so that it's easier to see 👍